File: 1758 minimum-changes-to-make-alternating-binary-string/solution.go
func minOperations(s string) int {
    ans:=0
    for i,v := range s {
        if ((i&1)==0 && v=='1') || ((i&1)==1 && v=='0') {
            ans++
        }
        
    }
    return min(ans,len(s)-ans)
}

File: 1758 minimum-changes-to-make-alternating-binary-string/solution.rs
impl Solution {
    pub fn min_operations(s: String) -> i32 {
        let mut ans = 0;
        let by = s.as_bytes();
        let n = by.len();
        for i in 0..n {
            if ((i&1)==0 && by[i] == b'1') || ((i&1)==1 && by[i] == b'0') {
                ans+=1;
            }
        }
        ans.min(n-ans) as i32
    }
}
